### PR TITLE
Remove CDO.hip_chat_log_room reference. Add CDO.slack_logging.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: lib/cdo/chat_client.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def self.developers(message, options={})
   end
 
   def self.log(message, options={})
-    message(CDO.hip_chat_log_room, message, options)
```
Comment (ashercodeorg) [inline lib/cdo/chat_client.rb:20]: It seems unnecessary for this to be specified by a `CDO` value, I see no reason why it shouldn't be hard-coded.

Comment (ashercodeorg) [reply lib/cdo/chat_client.rb:20]: TODO(asher): Remove `CDO.hip_chat_log_room`, `CDO.hip_chat_logging`, and `Slack.CHANNEL_MAP`.

File: lib/cdo/chat_client.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def self.developers(message, options={})
...
+    message("infra-#{ENV['RACK_ENV']}", message, options)
```
Comment (aoby) [inline lib/cdo/chat_client.rb:20]: nice

File: deployment.rb [hunk 3]
```diff
@@ -91,6 +91,7 @@ def load_configuration
     'ruby_installer'              => rack_env == :development ? 'rbenv' : 'system',
     'root_dir'                    => root_dir,
     'section_projects'            => [:staging, :adhoc, :development].include?(rack_env),
+    'slack_logging'               => false,
```
Comment (aoby) [inline deployment.rb:94]: Is this set to true for non-dev environments via chef?

Comment (ashercodeorg) [reply deployment.rb:94]: Nope, at least not yet. But I also didn't see where `hip_chat_logging` is set anywhere else.

**Comment by ashercodeorg** [comment]
Closing, to break this into two PRs, one for each constant.

**Pull request closed without merging**